app/src/main/java/com/allan/boardbuddies/fragments/NotesFragment.java

```diff
@@ -0,0 +1,67 @@
+package com.allan.boardbuddies.fragments;
+
+import android.os.Bundle;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.allan.boardbuddies.Note;
+import com.allan.boardbuddies.NoteAdapter;
+import com.allan.boardbuddies.R;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.google.android.material.snackbar.Snackbar;
+
+import java.util.ArrayList;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link NotesFragment newInstance} factory method to
```

Comment:
I don't see a newInstance method in these classes, remove comment?

Reply:
I don't really understand but it came with the autogenerated class, I will remove i guess